cli/dev/servicepolicy.go

```diff
@@ -6,10 +6,17 @@ import (
 )
 
 const SERVICE_POLICY_FILE = "service.policy.json"
+const (
+	PROP_SVC_URL     = "openhorizon.service.url"     // The unique name of the service.
+	PROP_SVC_NAME    = "openhorizon.service.name"    // The unique name of the service.
+	PROP_SVC_ORG     = "openhorizon.service.org"     // The multi-tenant org where the service is defined. If service.url is specified but this property is omitted, the org defaults to the org of the node, service or policy which is referring to the service.
+	PROP_SVC_VERSION = "openhorizon.service.version" // The version of a service using the same semantic version syntax.
+	PROP_SVC_ARCH    = "openhorizon.service.arch"    // The hardware architecture of the node this service can run on.
+)
```

Comment:
These are defined in externalpolicy.builtin_properties.go . Please do not redefine them.

Reply:
hm, those are removed from the code already